cybered/shared/src/cyberedappconfig.py

```diff
@@ -4,24 +4,66 @@
 
 
 class CyberEdAppConfig(AppConfig, metaclass=ABCMeta):
-    """An educational module for the CyberEd website."""
+    """An educational module for the CyberEd website.
+
+    An educational module is a linear sequence of pages. A page might contain instructional
+    materials, examples, and preferably some kind of interaction to reduce boredom. It is highly
+    recommended that each module implement at least one standalone interactive tool that a student
+    can repeatedly (ab)use without preventing them from moving on with the instructional content.
+
+    The information in this AppConfig is used to automatically discover new modules and add them
+    to the landing page. Each derived application must define a module name, base link, start link,
+    and description, in addition to whatever definitions AppConfig requires.
+    """
+
+    # NOTE: This attribute is required by the AppConfig base class, but we redundantly require it
```

Comment:
It feels wrong to redefine the abstract attribute `name`, but it confused me the first time I saw that there was a `name` and a `module_name`, so I wanted to document the difference between the two. Do you have a preference?

Reply:
Maybe instead of module_name it should be display_name? Then I think it kind of implies the difference? But I do think it's good to have both documented for the sake avoiding future confusion

Reply:
We could just put in a comment that says 'this should not be confused with name, which needs to be defined within the django AppConfig type. Is it any different now that I made this a mixin instead of inheriting AppConfig directly?